po/nb.po

```diff
@@ -7607,13 +7610,12 @@ msgstr ""
 #. like gb:SEDOL, de:WKN, ch:Valorennummer, fr:SICOVAM ...
 #. See http://en.wikipedia.org/wiki/ISIN for hints.
 #: gnucash/gnome-utils/dialog-commodity.c:173
-#, fuzzy
 msgid ""
 "\n"
 "Exchange code (ISIN, CUSIP or similar): "
 msgstr ""
 "\n"
-"Børskode (CUSIP eller lignende):"
+"Børskode (ISIN, CUSIP eller lignende): "
```

Comment:
ask Verdipapirsentralen (VPS) ASA, if a norwegian variant exists.

Reply:
There is https://www.isin.org/norway-isin/ for what imagine is foreign citizens.
It is marked as needing edit in Weblate.

Reply:
The ISIN is the international SIN, CUSIP is the national SIN in US, WKN in DE … If no NO variant exists, I would remove ", CUSIP".